entities/src/user.rs

```diff
@@ -9,8 +8,7 @@ use serde::{Deserialize, Serialize};
 pub struct Model {
     #[sea_orm(primary_key)]
     pub id: u64,
-    #[sea_orm(unique)]
-    #[serde(serialize_with = "serialization::vec_as_address")]
+    #[sea_orm(column_type = "Binary(BlobSize::Blob(Some(20)))", unique)]
```

Comment:
again a Vec<u8>, no need to change with Address? also is the serialization decorator intended